src/types/aws/ap/donations.ts

```diff
@@ -57,5 +57,5 @@ export type DonationsQueryParams = {
   viaId?: string;
   symbol?: string;
   asker: number | string;
-  status?: "final" | "pending";
+  status?: "final" | "pending" | "intent";
```

Comment:
```suggestion
  status?: "final" | "pending";
```
I believe this is not live yet? aside, PR looks good overall

Reply:
Actually we don't need this for this PR, will remove.

Reply:
Removed https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2873/commits/51012591745bac8b3f67912593384070074f1ebd